docs/musescoreCreation.md

```diff
@@ -26,7 +27,7 @@ Alternatively, notes in a chord can be played sequentially in a channel if the c
 Tuplets are currently unsupported. They can be faked by alternating between lengths every other note.
 For example, ` 1/4 - 1/4 - 1/4` in a triplet could be `1/8 - 1/4 - 1/8`.
 
-Tied notes are supported. However, **any tie lasting longer than a whole note is wholly unsupported**.
+Tied notes are supported. However, **any tie lasting longer than a whole note is wholly unsupported at this time**.
```

Comment:
Added because I found some odd behavior in certain musicxml exports that hasn't been documented. This odd behavior likely also affects small notes as well, like `1/32`